CONTRIBUTING.md

```diff
@@ -48,18 +83,6 @@ cd test/v200/apiTest
 go test -v
 ```
 
-### Pull Requests
-
-All commits must be signed off with the footer:
-
-```git
-Signed-off-by: Firstname Lastname <email@email.com>
-```
-
-Once you set your user.name and user.email in your git config, you can sign your commit automatically with git commit -s. When you think the code is ready for review, create a pull request and link the issue associated with it.
-
-Owners of the repository will watch out for and review new PRs.
-
-If comments have been given in a review, they have to be addressed before merging.
+# Contact us
 
-After addressing review comments, don’t forget to add a comment in the PR afterward, so everyone gets notified by Github and knows to re-review.
+If you have any questions, please visit us the `#devfile` channel under the [Kubernetes Slack](https://slack.k8s.io) workspace.
```

Comment:
https://kubernetes.slack.com/archives/C02SX9E5B55

Reply:
@yangcao77 @thepetk How is this change? https://github.com/devfile/api/pull/1597/commits/aa7e5b38212f2c766d9e9cbb1758cfde3a37c4c6